app/User.php

```diff
@@ -30,4 +30,12 @@ class User extends Authenticatable
 
     protected $test = 'testtest';
     protected $test2 = 'test2test2';
+
+    private $check_WIP = true;
+    private $check_WIP1 = true;
+    private $check_WIP2 = true;
+
+    private function testSkipWIP () {
+        return $check_WIP;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Review the data-flow - use of uninitialized value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://relisonar.relipasoft.com:9000/coding_rules#rule_key=php%3AS836)